### PR TITLE
Add clear edit log management command #9788

### LINKED ISSUE #9788: Allow command line management of edit_log entries
There doesn't appear to be a way to manipulate the edit_log list of record changes. However when an instance of Arches is cleared of all its records, it would be good to be able to also zero out the edit_log.

In particular, a CLI utility that just called `DELETE FROM edit_log;` would seem to suffice? A flag on the remove_resources operation would also be valuable to do it in one step.

### PR DESCRIPTION
**Pull request by johnatawnclementawn**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Add a management command to purge the edit_log

### Issues Solved

Resolves #9788 

### TODO:
- [x] Add corresponding documentation PR to update CLI reference docs

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: LUX 
*   Found by: @azaroth42 
*   Tested by: @apeters  
*   Designed by: @johnatawnclementawn

### CODE REVIEW

File: arches/management/commands/resources.py [hunk 1]
```diff
@@ -44,10 +44,21 @@ def add_arguments(self, parser):
...
+            "-e",
+            "--editlog",
+            action="store_true",
+            dest="editlog",
+            help="used to clear the edit log",
+        )
+
     def handle(self, *args, **options):
         if options["operation"] == "remove_resources":
             self.remove_resources(force=options["yes"], graphid=options["graph"])
```
Comment (apeters) [inline arches/management/commands/resources.py:57]:
```suggestion
            self.remove_resources(force=options["yes"], graphid=options["graph"], clear_edit_log=options["editlog"])
```

File: arches/management/commands/resources.py [hunk 2]
```diff
@@ -44,10 +44,21 @@ def add_arguments(self, parser):
...
 
+        if options["operation"] == "clear_edit_log":
+            self.clear_edit_log()
+
     def remove_resources(self, load_id="", graphid=None, force=False):
```
Comment (apeters) [inline arches/management/commands/resources.py:62]:
```suggestion
    def remove_resources(self, load_id="", graphid=None, force=False, clear_edit_log=False):
```

File: arches/management/commands/resources.py [hunk 3]
```diff
@@ -71,3 +82,10 @@ def remove_resources(self, load_id="", graphid=None, force=False):
             graph.delete_instances(verbose=True)
```
Comment (apeters) [inline arches/management/commands/resources.py:82]:
```suggestion
            graph.delete_instances(verbose=True)
            if clear_edit_log:
                self.clear_edit_log(graphid)
```

File: arches/management/commands/resources.py [hunk 4]
```diff
@@ -71,3 +82,10 @@ def remove_resources(self, load_id="", graphid=None, force=False):
...
 
         return
+    
+    def clear_edit_log(self):
```
Comment (apeters) [inline arches/management/commands/resources.py:86]:
```suggestion
    def clear_edit_log(self, graphid=None):
```

File: arches/management/commands/resources.py [hunk 5]
```diff
@@ -71,3 +82,10 @@ def remove_resources(self, load_id="", graphid=None, force=False):
...
+        """
+        Clears the edit log
+        """
+        models.EditLog.objects.all().delete()
```
Comment (apeters) [inline arches/management/commands/resources.py:90]:
```suggestion
        if graphid:
            models.EditLog.objects.filter(resourceclassid=graphid).delete()
        else:
            models.EditLog.objects.all().delete()
```

**Review by apeters — CHANGES_REQUESTED** [review]
passing in the `-e` flag when removing resources doesn't actually remove the edit-log entries

**Comment by johnatawnclementawn** [comment]
> dae5660

Fixed in dae566037841e5a59bdc1a4e4cd06c8bc44d4860

File: arches/management/commands/resources.py [hunk 6]
```diff
@@ -44,11 +44,22 @@ def add_arguments(self, parser):
...
+            dest="editlog",
+            help="used to clear the edit log. If a graphid is provided, only the edit log for that graph will be cleared.",
+        )
+
     def handle(self, *args, **options):
         if options["operation"] == "remove_resources":
-            self.remove_resources(force=options["yes"], graphid=options["graph"])
+            self.remove_resources(force=options["yes"], graphid=options["graph"], clear_edit_log=options["editlog"])
+
+        if options["operation"] == "clear_edit_log":
```
Comment (apeters) [inline arches/management/commands/resources.py:59]: you should be able to call the clear_edit_log command by itself by also passing in the `-g` flag to target just log entries for that graph

**Review by apeters — CHANGES_REQUESTED** [review]

File: arches/management/commands/resources.py (hunk 6 above)
Comment (johnatawnclementawn) [reply arches/management/commands/resources.py:59]: Fixed in [4be40d4](https://github.com/archesproject/arches/pull/10837/commits/4be40d4d46a409d12faf10ea11f21438642572ef)

**Pull request merged**